2016/opencv_lib/2702_proc.cpp

```diff
@@ -9,20 +9,38 @@ using namespace cv;
 
 using namespace std;
 
+Mat outputImage;
```

Comment:
Instead of having outputImage at a global scope (where you've declared it: that's global scope, meaning any function can freely access it), you should either return it from the function creating it like this:
```
Mat fixImage(Mat inputImage)
{
   Mat outputImage
   ... analysis steps that create outputImage
   return outputImage;
}